project/RemoveTestFromPomPlugin.scala

```diff
@@ -0,0 +1,60 @@
+import sbt.Keys.{pomPostProcess, publishArtifact, sLog}
```

Comment:
does `headerCreateAll` work also for files in `project/`?

Reply:
Nope :(

`sbt-header` doesn't work for files inside `project` folder

Reference: https://github.com/sbt/sbt-header/issues/65